btree/btree.c

```diff
@@ -2357,11 +2362,55 @@ static bool ff_rec_is_valid(const struct slot *slot)
 	   _0C(val_is_valid);
 }
 
+static bool ff_iskey_smaller(const struct nd *node, int prev_key_idx,
```

Comment:
The function is checking record at current jey index with next key index. It will be better to have second parameter name as `cur_key_idx`

Reply:
made this change.